src/com/company/EnemyTank.java

```diff
@@ -7,18 +7,78 @@
  * Created by akupp_000 on 5/8/2017.
  */
 public class EnemyTank extends Tank {
+	
+	private Handler handler;
+	
     public EnemyTank(float x, float y, ObjectId id, Handler handler){
         super(x,y,id, handler);
+        this.handler = handler;
     }
     @Override
     public void tick(LinkedList<GameObject> object) {
-
+    	Collision(object);
     }
-
+    
+    public void Collision(LinkedList<GameObject> object) {
+    	for(int i=0; i < handler.object.size();i++)
+    	{
+    		GameObject tempObject = handler.object.get(i);
+    		
+    		if(tempObject.getID() == ObjectId.Block)
+    		{
+    			if(getBoundsTop().intersects((tempObject.getBounds())))
+    			{
+    				y = tempObject.getY() + (height/2);
+    				velY= 0;
+    			}
+    			
+    			if(getBounds().intersects((tempObject.getBounds())))
+    			{
+    				y = tempObject.getY() - height;
+    				velY= 0;
+    			}	
+    			
+    			if(getBoundsRight().intersects((tempObject.getBounds())))
+    			{
+    				x = tempObject.getX() - width;
+    			}
+    			
+    			if(getBoundsLeft().intersects((tempObject.getBounds())))
+    			{
+    				x = tempObject.getX() - width;
+    			}
+    		}
+    	}
+    }
+    
     @Override
     public void render(Graphics g) {
         g.setColor(Color.RED);
         g.fillRect((int) x, (int) y, (int)width, (int)height);
         gun.render(g);
+        
+        Graphics2D g2d = (Graphics2D)g;
+        g.setColor(Color.black);
+        g2d.draw(getBounds());
+        g2d.draw(getBoundsRight());
+        g2d.draw(getBoundsLeft());
+        g2d.draw(getBoundsTop());
+    }
+    
+    public Rectangle getBounds(){
+    	return new Rectangle((int) ((int)x + (width/2)-((width/2)/2)), (int) ((int)y+(height/2))+5, (int)width/2, (int)(height/2)-10);
```

Comment:
IMO `(width/2)-((width/2)/2)` is equal to `((width/2)/2)`. Please consider replace to reduce performing calculations.